### PR TITLE
Improvements for RTKNAVI

### PR DESCRIPTION
**Pull request by ebethon**
Add support for the following functionality to RTKNAVI:
1. Support for Wide Lane Ambiguity Resolution
2. Support for Wide Lane Ambiguity Resolution with hold and fix
3. Resize options window for larger AR mode labels

### CODE REVIEW

**Review by dgburr — APPROVED** [review]
LGTM

File: app/winapp/rtknavi/naviopt.cpp [hunk 1]
```diff
@@ -937,6 +957,11 @@ void __fastcall TOptDialog::SaveOpt(AnsiString file)
 	prcopt.posopt[5]=PosOpt6->Checked;
 	
 	prcopt.modear	=AmbRes		->ItemIndex;
+	prcopt.wlmodear=0;
```
Comment (tao-swift) [inline app/winapp/rtknavi/naviopt.cpp:960]: same here.

File: app/winapp/rtknavi/naviopt.cpp [hunk 2]
```diff
@@ -533,6 +541,11 @@ void __fastcall TOptDialog::SetOpt(void)
 	PrcOpt.tropopt   =TropOpt     ->ItemIndex;
 	PrcOpt.sateph    =SatEphem    ->ItemIndex;
 	PrcOpt.modear    =AmbRes      ->ItemIndex;
+	PrcOpt.wlmodear=0;
```
Comment (tao-swift) [inline app/winapp/rtknavi/naviopt.cpp:544]: nit, this line doesn't seem to be needed. prcopt.wlmodear has been initialised as 0 in https://github.com/swift-nav/RTKLIB/blob/fcb4097b272cc682a71f20e2bb5a31e2e098081d/src/rtkcmn.c#L207

**Review by tao-swift — APPROVED** [review]
lgtm!

**Review by silverjam — APPROVED** [review]

**Review by silverjam — CHANGES_REQUESTED** [review]
Can we rebase against master and get a clean set of checks?  Latest master should remove Travis so that failure should go away once this is rebased.

**Review by silverjam — APPROVED** [review]

**Pull request merged**